workflows/nanopore.nf

```diff
@@ -135,6 +142,15 @@ workflow NANOPORE {
     )
     ch_versions = ch_versions.mix(SOURMASH_PROFILE_ASSEMBS.out.versions)
 
+    // run DIAMOND blastp of predicted proteins against db
+    DIAMOND_BLASTP (
+        ch_proteins,
+        ch_diamond_db,
+        "txt",
+        params.diamond_columns
```

Comment:
I assume this should also use `val_diamond_columns`

Reply:
that's what the params.diamond_columns is referring to, I've made that something the user has to input

Reply:
addressed